src/lib.zig

```diff
@@ -10,7 +10,7 @@ const hashes_of_zero = @import("./zeros.zig").hashes_of_zero;
 const Allocator = std.mem.Allocator;
 
 /// Number of bytes per chunk.
-const BYTES_PER_CHUNK = 32;
+pub const BYTES_PER_CHUNK = 32;
```

Comment:
same as the others, don't export it outside of the main module. If you need this in other module, define a `params.zig` file, that you can import from inside the other modules, but this should not be exported to the wider world